pyspectral/near_infrared_reflectance.py

```diff
@@ -91,6 +90,9 @@ def __init__(self, platform_name, instrument, band, **kwargs):
         if self.solar_flux is None:
             self._get_solarflux()
 
+        # The sun-zenith angle limit in degrees defining how far towards the
+        # terminator we try derive a
+        self.sunz_threshold = kwargs.get('sunz_threshold', TERMINATOR_LIMIT)
```

Comment:
Why no have `sunz_threshold` (along with `solar_flux` and `detector`) as explicit kwargs? it would make the code easier to follow.

Reply:
@mraspaud ? I thought I did.
`self.sunz_threshold = kwargs.get('sunz_threshold', TERMINATOR_LIMIT)`
How do you mean?

Reply:
I mean changing this:
```python
def __init__(self, platform_name, instrument, band, **kwargs):
```
to this
```python
def __init__(self, platform_name, instrument, band, sunz_threshold=None, solar_flux=None, detector=None):
```
It makes it easier to read the code...

Reply:
Ok, yes, I see. Will fix...